plugins/UserId/tests/System/expected/test_userIdNotOverwritesVisitorId___UserId.getUsers_range.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I'm noticing these test results are exactly the same as the normal `testApi()` tests. I think the Fixture needs to be changed, tracking the same visits w/ the new setting set. Maybe into a new site, then running API tests for the new site.

Reply:
👍  not sure what I was thinking @diosmosis . I've created a new fixture and new tests that test this behaviour now properly